product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -9,33 +9,31 @@ class TestProductSupplierinfo(common.SavepointCase):
     @classmethod
     def setUpClass(cls):
         super(TestProductSupplierinfo, cls).setUpClass()
-        cls.partner = cls.env['res.partner'].create({
+        cls.partner_obj = cls.env['res.partner']
+        cls.partner = cls.partner_obj.create({
             'name': 'Partner Test',
         })
-        cls.supplier1 = cls.env['res.partner'].create({
+        cls.supplier1 = cls.partner_obj.create({
             'name': 'Supplier #1',
         })
-        cls.supplier2 = cls.env['res.partner'].create({
+        cls.supplier2 = cls.partner_obj.create({
             'name': 'Supplier #2',
         })
-        cls.product = cls.env['product.product'].create({
-            'name': 'Product Test',
-            'seller_ids': [
-                (0, 0, {
-                    'name': cls.supplier1.id,
-                    'min_qty': 5,
-                    'price': 50,
-                }),
-                (0, 0, {
-                    'name': cls.supplier2.id,
-                    'min_qty': 1,
-                    'price': 10,
-                }),
-            ],
-        })
+        cls.product = cls.env.ref('product.product_product_2')
```

Comment:
Please don't use demo data. And why are you changing it having already without demo data?